cpp/oneapi/dal/table/test/table_adapter.cpp

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
```

Comment:
It should be placed near the functionality it is testing. Suggest moving it in `oneapi/dal/backend/interop/test` and name as `convert_to_daal_table` tests.

Reply:
It isn't possible because of cyclical dependence between table backend and common backend folders, we tried it with rlnx and defined other possible place for this test.